### PR TITLE
{chem}[intel/2017b] spglib v1.9.9

### PR DESCRIPTION
**Pull request by dquigley-warwick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by dquigley-warwick** [comment]
Test report by @dquigley-warwick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cedar.theory.warwick.ac.uk - Linux centos linux 7.5.1804, Intel(R) Core(TM)2 Quad CPU    Q8400  @ 2.66GHz, Python 2.7.5
See https://gist.github.com/cbbe2e49516d879752ae47760c29b867 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.11
See https://gist.github.com/73a6d6c6cdc60b94941e371dcef14bda for a full test report.

File: easybuild/easyconfigs/s/spglib/spglib-1.9.9-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+separate_build_dir = True
+
+# Build places libs and headers in the source directory - copy them manually
+installopts = ' && cp -r ../%(name)s-%(version)s/lib ../%(name)s-%(version)s/include %(installdir)s/'
+
+parallel = 1
+
+sanity_check_paths = {
+    'files': ['lib/libsymspg.so.%(version)s', 'lib/libsymspg.a'],
```
Comment (verdurin) [inline easybuild/easyconfigs/s/spglib/spglib-1.9.9-intel-2017b.eb]: @dquigley-warwick - can you explain the change here? Normally we use `SHLIB_EXT`, as it was in the older versions.

Comment (dquigley-warwick) [reply easybuild/easyconfigs/s/spglib/spglib-1.9.9-intel-2017b.eb]: No (good) reason. I'll update the PR.

**Comment by dquigley-warwick** [comment]
Test report by @dquigley-warwick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cedar.theory.warwick.ac.uk - Linux centos linux 7.5.1804, Intel(R) Core(TM)2 Quad CPU    Q8400  @ 2.66GHz, Python 2.7.5
See https://gist.github.com/ee970e38cde5be716a8e2adae60bb4a7 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @dquigley-warwick!

**Pull request merged**